css/night-owl.css

```diff
@@ -1,135 +1,173 @@
 .hljs {
-    display: block;
-    overflow-x: auto;
-    padding: 0.5em;
-    background: #011627;
-    color: #d6deeb;
+  display: block;
+  overflow-x: auto;
+  padding: 0.5em;
+  background: #011627;
+  color: #d6deeb;
 }
 .hljs-keyword {
-    color: #c792ea;
-    font-style: italic;
+  color: #c792ea;
+  font-style: italic;
+  background: #011627;
 }
 .hljs-built_in {
-    color: #addb67;
-    font-style: italic;
+  color: #addb67;
+  font-style: italic;
+  background: #011627;
 }
 .hljs-type {
-    color: #82aaff;
+  color: #82aaff;
+  background: #011627;
 }
 .hljs-literal {
-    color: #ff5874;
+  color: #ff5874;
+  background: #011627;
 }
 .hljs-number {
-    color: #f78c6c;
+  color: #f78c6c;
+  background: #011627;
 }
 .hljs-regexp {
-    color: #5ca7e4;
+  color: #5ca7e4;
+  background: #011627;
 }
 .hljs-string {
-    color: #ecc48d;
+  color: #ecc48d;
+  background: #011627;
 }
 .hljs-subst {
-    color: #d3423e;
+  color: #d3423e;
+  background: #011627;
 }
 .hljs-symbol {
-    color: #82aaff;
+  color: #82aaff;
+  background: #011627;
 }
 .hljs-class {
-    color: #ffcb8b;
+  color: #ffcb8b;
+  background: #011627;
 }
 .hljs-function {
-    color: #82aaff;
+  color: #82aaff;
+  background: #011627;
 }
 .hljs-title {
-    color: #dcdcaa;
-    font-style: italic;
+  color: #dcdcaa;
+  font-style: italic;
+  background: #011627;
 }
 .hljs-params {
-    color: #7fdbca;
+  color: #7fdbca;
+  background: #011627;
 }
 .hljs-comment {
-    color: #637777;
-    font-style: italic;
+  color: #637777;
+  font-style: italic;
+  background: #011627;
 }
 .hljs-doctag {
-    color: #7fdbca;
+  color: #7fdbca;
+  background: #011627;
 }
```

Comment:
variables can be used for color and background

Reply:
Ack, Thanks for the suggestion, will implement